src/Sulu/Bundle/AdminBundle/ResourceMetadata/Datagrid/Field.php

```diff
@@ -28,6 +28,21 @@ class Field
      */
     protected $type;
 
+    /**
+     * @var string
+     */
+    protected $visibility;
+
+    /**
+     * @var bool
+     */
+    protected $sortable;
```

Comment:
I think I would also remove that as long as it is not implemented... And btw, implementing it wouldn't be too much work I guess, right? Maybe that's also something we can get done :smiley: